lib/_http_outgoing.js

```diff
@@ -573,6 +573,7 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
   if (!msg._hasBody) {
     debug('This type of response MUST NOT have a body. ' +
           'Ignoring write() calls.');
+    if (callback) process.nextTick(callback);
```

Comment:
We could replace `chunk` with an empty chunk and continue without bailing out, but I think it makes sense to short circuit here as **all** chunks are ignored.